### DISCUSSION TITLE
Allure report tab is not displayed in Azure Release/Classic pipelines

### DISCUSSION THREAD

**Discussion by suganyavasudev** [allure-framework/allure2#2763] [Bug Reports]
using extension of qameta for allure report ( https://marketplace.visualstudio.com/items?itemName=qameta.allure-azure-pipelines) , we are not able to view allure report tab in release/classic pipeline though Publish allure report task is successful . Due to which we need to drop allure report in our project, If we get a fix to support azure classic pipeline too would be better solution

![classicpipeline](https://github.com/user-attachments/assets/f569ecb6-1ec6-41b5-87cf-83fe51020a17)

**Comment by gabrielaiobogeanu** [comment]
hey, can you please share your configuration file  ?

**Reply by suganyavasudev** [reply]
![image](https://github.com/user-attachments/assets/c8d88b1a-32ce-494e-9916-7e026e4be946)

**Comment by gabrielaiobogeanu** [comment]
can you please also provide the yaml file for java installation ?

**Reply by suganyavasudev** [reply]
This is classic pipeline, please check the below details of java installation
![image](https://github.com/user-attachments/assets/c2acfb12-3804-48ef-8e51-db88bcd3276e)
![image](https://github.com/user-attachments/assets/7092883b-dfae-4b4c-bfe4-a4e35c1fee6b)
